### ISSUE TITLE
[Documentation] Multiple package.json explanation?

### ISSUE THREAD

**Issue by dalepo** [nxtend-team/nxtend#410]
Hello, thanks for this plugin. 

I'm trying to get to know how it works but I feel like it should be an explanation on why new `ionic-angular` have multiple packages.json files.

Adding new `ionic-angular` app would result in a new package.json file under /apps/your-app-name/package.json
```
{
  "name": "ionic-application",
  "devDependencies": {
    "@capacitor/cli": "^2.4.5"
  }
}
```
Is the Ionic-app supposed to have new dependencies there or to just keep it to use the capacitor cli ?.

**Comment by devinshoemaker** [comment]
Yes, this is expected behavior. This is due to the fact the Capacitor must be ran within the project directory, and must have a `package.json` present with the Capacitor CLI installed in that folder. I do have this documented briefly, but I do think it could be improved. https://nxtend.dev/docs/capacitor/getting-started#adding-capacitor-plugins

**Comment by Radecom** [comment]
### Is it possible to work with a project with its own node_modules folder inside it while solving problems of incompatibility with monorepo of a package?

Hello, first of all, congratulations on your outstanding work, thanks to it many of us have been interested in moving our projects to Monorepositories.

It is normal to find some difficulties during the conversion process; for example, some dependencies of your project are not developed to work in a multi-project workspace; for example, I was transforming a multi-repo project that I have, and I ran into this error:

Error: Module not found: Error: Can't resolve '../../../../../capacitor.config.json'

For that reason, I had to stop my migration temporarily, but the project must continue, hence my question.

Is it possible to temporarily work with that project with its own node_modules folder inside, which fixes that problem?

More than a yes or no answer, is there a methodology to follow in such a situation?
